config/eslintTS.js

```diff
@@ -6,7 +6,9 @@ module.exports = {
     "prettier/@typescript-eslint",
     "plugin:prettier/recommended",
   ],
+  plugins: ["import", "react"],
   rules: {
+    "@typescript-eslint/prefer-readonly": "error",
```

Comment:
We don't need this rule as it's not affecting our code. Can you disable `prefer-readonly-parameter-types` for the problematic file using an ESLint disable directive or `overrides` field?